tests/stage_2/com_macro-if-1.txt

```diff
@@ -1,2 +1,2 @@
-brun -s2 '(opt (com (q . (if @ 999 2000)) (c (a (q (opt (com (q . (defmacro if (A B C) (qq (a (i (unquote A) (function (unquote B)) (function (unquote C))) @))))))) 1) (a (q (opt (com (q . (macros))))) 0))))'
+brun -s2 '(opt (com (q . (if @ 999 2000)) (c (a (opt (com (q . (defmacro if (A B C) (qq (a (i (unquote A) (function (unquote B)) (function (unquote C))) @)))))) 1) (a (opt (com (q . (macros)))) 0))))'
```

Comment:
This one I don't get. I understand the new code, in the sense that we expect `opt` to return the first arg for `a`

Why did we need to quote the old args to `a` ?

Reply:
I believe this test was originally written because I was having problems porting the code that compiles `if` from python to a clvm macro, and it just sort of evolved as semantics changed without completely understanding what it was really trying to do. I believe the version above probably works with any version of `tools` after the `(q . 0)` change. Some old version may have done an implicit `((...))` style call. Definitely many elements of this test suite could do with more comments as to what motivated them.